rizzino_a05a.js

```diff
@@ -0,0 +1,21 @@
+/* Julie Rizzino
+rizzino_a05a.js
+INFO 2124
+Thoendel
+1/14/20 */
+
+let toDoList = [`Fold laundry`,`Cook breakfast`,`Take a shower`,`Make my bed`];
+console.log(toDoList);
+toDoList.unshift(`Go grocery shopping`);
+console.log(toDoList);
+toDoList.reverse();
+toDoList.sort();
+console.log(toDoList);
+toDoList.includes(`x`);
```

Comment:
I'd love to see you do something with this - i.e. output that the list does or does not contain "X"